modules/consensus/consensusset.go

```diff
@@ -187,15 +191,10 @@ func New(gateway modules.Gateway, saveDir string) (*ConsensusSet, error) {
 		return nil, err
 	}
 
-	// During short tests, use an in-memory database.
-	if build.Release == "testing" && testing.Short() {
-		cs.db = persist.NilDB
-	} else {
-		// Otherwise, try to load an existing database from disk.
-		err = cs.load(saveDir)
-		if err != nil {
-			return nil, err
-		}
+	// Try to load an existing database from disk.
+	err = cs.load(saveDir)
```

Comment:
I'm a little surprised that the saveDir is not already in the consensus struct. I guess there's not an explicit need for it, but as we do more things like logging I think it would make sense. (even though, I think, it would still only be used at startup.

No need to do anything about it this PR, just an observation.